client/landing/stepper/declarative-flow/internals/steps-repository/pattern-assembler/survey/survey.tsx

```diff
@@ -0,0 +1,40 @@
+import { useLocale } from '@automattic/i18n-utils';
+import { useState } from 'react';
+import Banner from 'calypso/components/banner';
+import './survey.scss';
+
+const Survey = ( { userHasEngaged }: { userHasEngaged: boolean } ) => {
+	const locale = useLocale();
+	const [ dismissed, setDismissed ] = useState( false );
+
+	if ( 'en' !== locale || dismissed || ! userHasEngaged ) {
+		return null;
+	}
+
+	return (
+		<Banner
+			className="pattern-assembler__survey"
+			title={
+				<>
+					<a href="https://lucasmdo.survey.fm/assembler-survey-modal" target="blank">
+						Fill out this quick survey
+					</a>{ ' ' }
+					and help us to improve our product.
```

Comment:
We have to use `translate` with component args for the better translation 🙂

```js
translate( '{{a}}Fill out this quick survey{{/a}} and help us to improve our product.', { ... } );
```

Reply:
We are not translating here because we only target English users. cc: @taipeicoder

Reply:
I think it's fine to not translate the strings here, but let's leave a comment so that people know this is intended.

Reply:
Yep. Let's add comment to explain it's intended 🙂